moonraker/components/file_manager/file_manager.py

```diff
@@ -518,7 +518,17 @@ async def _finish_gcode_upload(self,
                 raise self.server.error(
                     "File is loaded, upload not permitted", 403)
         self.notify_sync_lock = NotifySyncLock(upload_info['dest_path'])
-        finfo = await self._process_uploaded_file(upload_info)
+        # If moving forward with the WIP idea, some of the functionality
+        # of this method will need to be reimplemented.  Also some
+        # additional error handling throughout this process
+        # finfo = await self._process_uploaded_file(upload_info)
+
+        finfo = self.get_path_info(
+            upload_info['tmp_file_path'], upload_info['root'])
+        if upload_info['unzip_ufp']:
+            finfo['ufp_path'] = upload_info['tmp_file_path']
+        else:
+            finfo['gcode_path'] = upload_info['tmp_file_path']
```

Comment:
This is going to break inotify functionality. When an inotify watch detects a new file in the gcode path its going to request the metadata parser to process the file at the existing location.  The combination of this change and the change to the metadata parser is going to result in a failure.